test/assert.js

```diff
@@ -365,7 +365,7 @@ describe('assert', function () {
 
     err(function () {
       assert.deepEqual(obj1, obj2);
-    }, "expected { tea: \'chai\' } to deeply equal { tea: \'black\' }");
+    }, "expected {} to deeply equal {}");
```

Comment:
I feel like we're losing data here. Perhaps this should be a feature of loupe that if the object has no own-properties then it can inspect the prototype to make it more useful for instances like this?